src/main/java/ua/com/juja/microservices/teams/slackbot/service/TeamService.java

```diff
@@ -0,0 +1,15 @@
+package ua.com.juja.microservices.teams.slackbot.service;
+
+import org.springframework.stereotype.Service;
+import ua.com.juja.microservices.teams.slackbot.model.Team;
+import ua.com.juja.microservices.teams.slackbot.model.TeamRequest;
+
+/**
+ * @author Ivan Shapovalov
+ */
+@Service
```

Comment:
As I know such annotation should be only above conrete classes but if I'm wrong - proove it.